Extension/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Lib.Harmony" version="2.0.0.9" targetFramework="net472" />
+  <package id="Lib.Harmony" version="2.0.0.10" targetFramework="net472" />
```

Comment:
This is added in another PR

Reply:
Will be resolved when harmony update is on master and merged into this branch